src/components/app/app.tsx

```diff
@@ -1,18 +1,22 @@
 import * as React from "react";
+import { Layout, Display, JustifyContent } from "ui";
 import "./app.scss";
 
 type AppProps = {
   message: string;
 };
 
-/**
- * Sample App
- */
 const App: React.FunctionComponent<AppProps> = (props: AppProps) => {
   return (
-    <div className={"container"}>
-      <h1 className={"header shifting-rainbow-text"}>{props.message}</h1>
-    </div>
+    <Layout
+      display={Display.Flex}
+      justifyContent={JustifyContent.Center}
+      className="container"
+    >
+      <Layout margin={{ x: 1 }} className="header">
```

Comment:
is it normal to use a `div` classed as a `header` wrapped around an `h1` header element? Or rather, is there a case against just sticking to one of them?

Reply:
in HTML5, you would normally use just a `<header>` tag, but generally yeah. A header should be a block element that contains information (like an `h1` element which is just a text element)